build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/AnalyticsUtils.java

```diff
@@ -0,0 +1,125 @@
+package io.quarkus.ts.buildtimeanalytics;
+
+import static io.quarkus.test.bootstrap.QuarkusCliClient.CreateApplicationRequest.defaults;
+
+import io.quarkus.test.bootstrap.QuarkusCliClient;
+import io.quarkus.test.services.quarkus.model.QuarkusProperties;
+
+public class AnalyticsUtils {
+    public static final String QUARKUS_ANALYTICS_DISABLED_PROPERTY = "quarkus.analytics.disabled";
+    public static final String QUARKUS_ANALYTICS_URI_BASE_PROPERTY = "quarkus.analytics.uri.base";
+    public static final String QUARKUS_ANALYTICS_TIMEOUT_PROPERTY = "quarkus.analytics.timeout";
+    public static final String QUARKUS_ANALYTICS_FAKE_URI_BASE = "http://fake.url";
+    public static final String QUARKUS_ANALYTICS_EVENT_DEV = "DEV_MODE";
+    public static final String QUARKUS_ANALYTICS_EVENT_PROD = "BUILD";
+    public static final String QUARKUS_ANALYTICS_IP = "0.0.0.0";
+    public static final String NON_EXCLUDED_GROUP_ID = "rh.quarkus.qe";
+    public static final String EXCLUDED_GROUP_ID = "io.quarkus.qe";
+    public static final String UNRECOGNIZED_PROPERTY_FORMAT = "Unrecognized configuration key \"%s\" was provided";
+    public static final String QUARKUS_SNAPSHOT_VERSION = "999-SNAPSHOT";
+    public static final String QUARKUS_EXTENSION_SNAPSHOT_VERSION_PATTERN = "999-.*";
```

Comment:
Why do we need this pattern? Isn't it always "999-SNAPSHOT"?

Reply:
Quarkus reports extensions' versions in the analytics payload as full artifact version. So for `999-SNAPSHOT`, it can look like e.g. `999-20230718.203351-1091`.

Reply:
Wow. Could you put this into comment?

Reply:
Will do.